atomics/include/desul/atomics/Generic.hpp

```diff
@@ -598,12 +610,31 @@ DESUL_INLINE_FUNCTION T atomic_fetch_inc(T* const dest,
   return atomic_fetch_add(dest, T(1), order, scope);
 }
 
+template <typename T, class MemoryOrder, class MemoryScope>
+DESUL_INLINE_FUNCTION T atomic_wrapping_fetch_inc(T* const dest,
+                                         T val,
+                                         MemoryOrder order,
+                                         MemoryScope scope) {
+  static_assert(std::is_unsigned<T>::value, "Signed types not supported by atomic_wrapping_fetch_inc.");
```

Comment:
Seems that one of the RAJA tests in particular uses signed types when using these functions. I think that the static restriction here makes sense, so I'll see about changing the test.

Reply:
That sounds good.  It was probably happening to work by accident before, I'd rather people get the error this way than find it when they use a counter that goes one bit too high.